### PR TITLE
prep/stable

### PR DESCRIPTION
**Pull request by vmelamed**
# Summary

## Commits

- docs: cosmetic
- chore: update changelog for v3.0.3-preview.2 [skip ci]
- feat: add .slnx support in .gitattributes and update vm2.TestUtilities to version 1.4.1 in package locks chore: add changelog reminder step in Prerelease workflow fix: update changelog parsers for documentation messages docs: add .gitmessage template for commit message guidelines
- chore: bump vm2.TestUtilities to 1.4.2 and align changelog parser
- fix: update tag_pattern regex to enforce semantic versioning format
- chore: update documentation regex for commit messages and add pull request template

## Checklist

- [x] Commit messages follow [Conventional Commits](https://www.conventionalcommits.org/) format
- [x] Tests pass locally, and cover new code and edge cases. Test coverage is above 80%.
- [x] It is not expected that the performance will degrade by more than 20%
- [x] Documentation is updated if necessary

### CODE REVIEW

File: changelog/cliff.prerelease.toml [hunk 1]
```diff
@@ -38,7 +37,7 @@ commit_parsers = [
     { message = "^refactor", group = "Internal" },
     { message = "^perf", group = "Performance" },
     { message = "^security", group = "Security" },
-    { message = "^docs", group = "Internal" },
+    { message = "^docs?(\\([^)]*\\))?!?:", group = "Internal" },
     { message = "^chore", group = "Internal" },
```
[the remark this answers is not in the document]
Comment (vmelamed) [reply changelog/cliff.prerelease.toml:41]: @copilot apply changes based on [this feedback](https://github.com/vmelamed/vm2.Ulid/pull/34#discussion_r3076659489)
